Tests/DependencyInjection/Compiler/CommandHandlerPassTest.php

```diff
@@ -63,58 +77,168 @@ public function testProcess()
-                    ['command' => 'my_command']
+            ]);
+
+        $this->container->shouldReceive('setDefinition')
+            ->with(
+                'tactician.commandbus.custom_bus.handler.locator',
+                \Mockery::type('Symfony\Component\DependencyInjection\Definition')
+            );
+
+        $this->container->shouldReceive('setDefinition')
+            ->with(
+                'tactician.commandbus.custom_bus.middleware.command_handler',
+                \Mockery::on(function (Definition $definition) {
+                    $methodNameInflectorServiceId = (string) $definition->getArgument(2);
+                   
+                    return $methodNameInflectorServiceId === 'tactician.handler.method_name_inflector.handle';
+                })
+            );
+
+        $this->container->shouldReceive('setAlias')
+            ->with(
+                'tactician.handler.locator.symfony',
+                'tactician.commandbus.custom_bus.handler.locator'
+            )
+            ->once();
+
+        $this->container->shouldReceive('setAlias')
+            ->with(
+                'tactician.middleware.command_handler',
+                'tactician.commandbus.custom_bus.middleware.command_handler'
+            )
+            ->once();
+
+        $this->compiler->process($this->container);
+    }
+
+    public function testProcessAddsHandlerDefinitionWithNonDefaultMethodNameInflector()
+    {
+        $definition = \Mockery::mock(Definition::class);
+
+        $this->container->shouldReceive('getExtensionConfig')
+            ->with('tactician')
+            ->once()
+            ->andReturn([
+                'default_bus' => 'custom_bus',
+                'method_inflector' => 'tactician.handler.method_name_inflector.handle_class_name',
+                'commandbus' => [
+                    'custom_bus' => []
+                ]
+            ]);
+
+        $this->container->shouldReceive('findTaggedServiceIds')
+            ->with('tactician.handler')
+            ->once()
+            ->andReturn([
+                'service_id_1' => [
+                    ['command' => 'my_command', 'bus' => 'custom_bus']
                 ],
             ]);
 
-        $definition->shouldReceive('addArgument')
-            ->never();
+        $this->container->shouldReceive('setDefinition')
+            ->with(
+                'tactician.commandbus.custom_bus.handler.locator',
+                \Mockery::type('Symfony\Component\DependencyInjection\Definition')
+            );
+
+        $this->container->shouldReceive('setDefinition')
+            ->with(
+                'tactician.commandbus.custom_bus.middleware.command_handler',
+                \Mockery::on(function (Definition $definition) {
+                    $methodNameInflectorServiceId = (string) $definition->getArgument(2);
+                   
+                    return $methodNameInflectorServiceId === 'tactician.handler.method_name_inflector.handle_class_name';
```

Comment:
This test has a lot of boilerplate (most of these tests seem to, maybe thats fine) but the real reason it exist is to confirm that we are using the method_inflector config value in the command handler.